[rrd4j] console commands openhab:rrd4j ... broken

<!-- Provide a general summary of the issue in the *Title* above -->
<!-- If the issue is related to a binding, please include its short name in -->
<!-- square brackets in the title - Example: "[astro] My issue..." -->

<!-- Important: Please contact the openHAB community forum for questions or -->
<!-- for configuration and usage guidance: https://community.openhab.org -->

<!-- Feel free to delete any comment lines in the template (starting with "<!--") -->

The console commands

openhab:rrd4j check
openhab:rrd4j clean

fail with 

Checking RRD files…
An unexpected error occurred during execution.

## Expected Behavior
<!-- If you're describing a bug, tell us what should happen -->
<!-- If you're suggesting a change/improvement, tell us how it should work -->

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->

~~~
20:21:13.959 [ERROR] [ab.core.io.console.ConsoleInterpreter] - An error occurred while executing the console command.
java.lang.IllegalStateException: stream has already been operated upon or closed
        at java.util.stream.AbstractPipeline.<init>(Unknown Source) ~[?:?]
        at java.util.stream.ReferencePipeline.<init>(Unknown Source) ~[?:?]
        at java.util.stream.ReferencePipeline$StatelessOp.<init>(Unknown Source) ~[?:?]
        at java.util.stream.ReferencePipeline$2.<init>(Unknown Source) ~[?:?]
        at java.util.stream.ReferencePipeline.filter(Unknown Source) ~[?:?]
        at org.openhab.persistence.rrd4j.internal.console.RRD4jCommandExtension.checkAndClean(RRD4jCommandExtension.java:132) ~[?:?]
        at org.openhab.persistence.rrd4j.internal.console.RRD4jCommandExtension.execute(RRD4jCommandExtension.java:87) ~[?:?]
        at org.openhab.core.io.console.ConsoleInterpreter.execute(ConsoleInterpreter.java:57) ~[?:?]
        at org.openhab.core.io.console.karaf.internal.CommandWrapper.execute(CommandWrapper.java:79) ~[?:?]
        at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:68) ~[?:?]
        at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:86) ~[?:?]
        at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:599) ~[?:?]
        at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:526) ~[?:?]
        at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:415) ~[?:?]
        at org.apache.felix.gogo.runtime.Pipe.doCall(Pipe.java:416) ~[?:?]
        at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:229) ~[?:?]
        at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:59) ~[?:?]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
        at java.lang.Thread.run(Unknown Source) [?:?]

~~~

This is from OH 5.0.3.

It has been reported also on the forum, https://community.openhab.org/t/openhab-5-1-milestone-discussion/166385/271

## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug, -->
<!-- or ideas how to implement the addition or change -->


## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->

open cli console
openhab:rrd4j check

## Context
<!-- How has this issue affected you? What are you trying to accomplish? -->
<!-- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used: (e.g., openHAB and add-on versions)
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):

--- a/bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/console/RRD4jCommandExtension.java
+++ b/bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/console/RRD4jCommandExtension.java
@@ -17,10 +17,9 @@ import java.nio.file.Path;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
-import java.util.Map.Entry;
+import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
@@ -114,8 +113,7 @@ public class RRD4jCommandExtension extends AbstractConsoleCommandExtension imple
 
         PersistenceServiceConfiguration config = persistenceServiceConfigurationRegistry
                 .get(RRD4jPersistenceService.SERVICE_ID);
-        Stream<Entry<String, String>> aliases = config != null ? config.getAliases().entrySet().stream()
-                : Stream.empty();
+        Map<String, String> aliases = config == null ? Map.of() : Map.copyOf(config.getAliases());
 
         console.println((checkOnly ? "Checking" : "Cleaning") + " RRD files...");
         int nb = 0;
@@ -128,8 +126,8 @@ public class RRD4jCommandExtension extends AbstractConsoleCommandExtension imple
                 boolean itemFound;
                 try {
                     // Map alias back to item
-                    String item = Objects.requireNonNull(
-                            aliases.filter(e -> name.equals(e.getValue())).findAny().map(e -> e.getKey()).orElse(name));
+                    String item = Objects.requireNonNull(aliases.entrySet().stream()
+                            .filter(e -> name.equals(e.getValue())).findAny().map(e -> e.getKey()).orElse(name));
                     itemRegistry.getItem(item);
                     itemFound = true;
                 } catch (ItemNotFoundException e) {

